src/viam/app/app_client.py

```diff
@@ -278,12 +280,58 @@ def proto(self) -> LogEntryPB:
         )
 
 
+class FragmentVisibility(str, Enum):
+    """
+    FragmentVisibility specifies who is permitted to view the fragment.
+    """
+
+    PRIVATE = "private"
+    """
+    Only visible to members in the fragment's organization.
+    """
+
+    PUBLIC = "public"
+    """
+    Visible to anyone and appears on the fragments page.
+    """
+
+    PUBLIC_UNLISTED = "public_unlisted"
+    """
+    Visible to anyone but does not appear on the fragments page.
+    """
+
+    UNSPECIFIED = "unspecified"
+    """
+    Uninitialized visibility.
+    """
```

Comment:
I would move this class to be a nested class within `Fragment` and rename it to just `Visibility`